kotlin-checks-test-sources/src/main/kotlin/checks/MergeIfElseIntoWhenCheckSample.kt

```diff
@@ -0,0 +1,167 @@
+package checks
+
+class MergeIfElseIntoWhenCheckSample {
+
+    fun returnOneIfCompliant(value: Int): String {
+        return if (value > 0) { // Complaint
+            "positive"
+        } else {
+            "negative"
+        }
+    }
+
+    fun returnOneIfNoElseCompliant(value: Int): String {
+        if (value > 0) { // Complaint
+            return "positive"
+        }
+        return "negative"
+    }
+
+    fun returnTwoIfNoncompliant(value: Int): String {
+        return if (value > 0) { // Noncompliant {{Merge chained "if" statements into a single "when" statement.}}
```

Comment:
you can also check issue location with comments:

```
return if (value > 0) {
//     ^^        
```

Reply:
Fixed.